lib/langchain/chatService.ts

```
@@ -24,25 +26,39 @@ export class ChatService {
       throw new Error("Invalid model selected.");
     }
   }
 
-  async sendMessage(message: string) {
+  public static getInstance(config: {
+    openAIKey?: string;
+    anthropicKey?: string;
+    selectedModel: ModelType;
+  }) {
+    const configHash = JSON.stringify(config);
+    if (configHash !== this.lastConfig && !this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)